crates/cxx-qt-gen/src/writer/cpp/header.rs

```diff
@@ -104,8 +104,8 @@ fn qobjects_header(generated: &GeneratedCppBlocks) -> Vec<String> {
         base_class = qobject.base_class,
         metaobjects = qobject.blocks.metaobjects.join("\n  "),
         methods = create_block("public", &qobject.blocks.methods.iter().map(pair_as_header).collect::<Vec<&str>>()),
-        slots = create_block("public Q_SLOTS", &qobject.blocks.slots.iter().map(pair_as_header).collect::<Vec<&str>>()),
-        signals = create_block("Q_SIGNALS", &qobject.blocks.signals.iter().map(AsRef::as_ref).collect::<Vec<&str>>()),
+        slots = create_block("public", &qobject.blocks.slots.iter().map(pair_as_header).collect::<Vec<&str>>()),
```

Comment:
and, eg, these would be removed as it would come from the `methods` line above etc